dashboard/test/models/pd/application/teacher_application_test.rb

```diff
@@ -963,6 +963,64 @@ class TeacherApplicationTest < ActiveSupport::TestCase
       assert application.reload.principal_approval_state.include? 'Complete - Admin said Yes on'
     end
 
+    test 'scholarship criteria for rp guardrail percents rp set fields' do
```

Comment:
I can't really understand what these are testing by the title –– for me, it's helpful to read the title as expected behavior, rather than the thing that's being tested (again, for `staging-next`)

Reply:
Oh good point, I'll make them clearer thank you!